.github/workflows/publish-react-native.yml

```
@@ -0,0 +1,52 @@
+name: Publish React Native Plugin
+on:
+  workflow_call:
+    inputs:
+      ref:
+        description: 'commit/tag/branch reference'
+        required: true
+        type: string
+      package-version:
+        description: 'version for the npm package (MAJOR.MINOR.BUILD)'
+        required: true
+        type: string
+      publish:
+        description: 'value indicating whether to publish to npm.'
+        required: true
+        type: boolean
+        default: false
+    secrets:
+      NPM_TOKEN:
+        description: 'access token for npm publish'
+        required: true
+
+jobs:
+  build-package:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout breez-sdk repo
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.ref || github.sha }}
+
+      - name: Setup node
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+          registry-url: 'https://registry.npmjs.org'
+          scope: '@breeztech'
+
+      - name: Make sure we publish the version as specified
+        working-directory: libs/sdk-react-native
+        run: npm --no-git-tag-version --allow-same-version version ${{ inputs.package-version }}
```

Do set a default version here, in case you want to test the build through a push command (in that case there are no input parameters.
```suggestion
        run: npm --no-git-tag-version --allow-same-version version ${{ inputs.package-version || '0.0.2' }}
```